soccer/main.cpp

```diff
@@ -114,6 +120,27 @@ int main(int argc, char* argv[]) {
             playbookFile = argv[++i];
         } else if (strcmp(var, "-noref") == 0) {
             noref = true;
+        } else if (strcmp(var, "-defend") == 0) {
+            if (i + 1 >= argc) {
+                printf("Field half not specified after -defend\n");
+                usage(argv[0]);
+            }
+            i++;
+            if (strcmp(argv[i], "plus") == 0) {
```

Comment:
Can you check for the alternate case here as well (and fail when we get an invalid argument). 

Failing is much better than assuming defaults silently, imo.